templates/gcp/fips.html

```diff
@@ -4,354 +4,446 @@
-      <p>Ubuntu Pro FIPS 18.04 LTS includes a Google Cloud-optimised FIPS-certified 4.15 kernel and other FIPS certified modules pre-enabled out of the box. Ubuntu Pro FIPS provides expanded security maintenance through April 2028.</p>
-      <p><a href="https://console.cloud.google.com/compute/" class="p-button--positive"><span>Launch now</span></a></p>
+</section>
+
+<section class="p-section">
+  <div class="row--50-50">
+    <hr />
+    <div class="col">
+      <h2>Launch Ubuntu Pro FIPS for Google Cloud</h2>
     </div>
-    <div class="col-6 p-divider__block">
-      <h4>Ubuntu Pro FIPS 20.04 LTS</h4>
-      <p>Ubuntu 20.04 LTS includes FIPS-certified 5.4  kernel and other FIPS certified modules pre-enabled out of the box. Ubuntu Pro FIPS provides expanded security maintenance through April 2030.</p>
-      <p><a href="https://console.cloud.google.com/compute/" class="p-button--positive"><span>Launch now</span></a></p>
+    <div class="col">
+      <h3 class="p-heading--5">Ubuntu Pro FIPS 20.04 LTS</h3>
+      <p class="p-section--shallow">Ubuntu 20.04 LTS includes FIPS-certified 5.4  kernel and other FIPS certified modules pre-enabled out of the box. Ubuntu Pro FIPS provides expanded security maintenance through April 2030.</p>
+      <hr class="is-muted" />
+      <ul class="p-inline-list">
+        <li class="p-inline-list__item">
+      <a href="https://console.cloud.google.com/compute/" class="p-button">Launch now</a>
+        </li>
+        <li class="p-inline-list__item">
+      <a href="https://console.cloud.google.com/compute/">18.04 LTS &nbsp;&rsaquo;</a>
+    </li>
+    <li class="p-inline-list__item">
+      <a href="https://console.cloud.google.com/compute/">16.04 LTS &nbsp;&rsaquo;</a>
+    </li>
     </div>
   </div>
 </section>
 
-<section class="p-strip">
-  <div class="row u-equal-height u-vertically-center">
-    <div class="col-12">
+<section class="p-section">
+  <div class="row--50-50">
+    <hr />
+    <div class="col">
       <h2>Google Cloud compliance made easy</h2>
-      <p><strong>Ubuntu Pro FIPS Google Cloud compliance made easy</strong></p>
-      <p>With Google Cloud, US government agencies and their partners have been able to combine unparalleled flexibility, breakthrough innovation, world class security and Google Cloud compliance. However when it comes to running a secure and compliant enterprise Linux on Google Cloud, IT departments are often left with expensive options that sacrifice innovation for compliance. Watch this video on demand to see how easy for government agencies to consume Linux on Google Cloud.</p>
-     <p><a href="https://ubuntu.com/blog/enable-fips-on-google-cloud" class="p-button--positive">Read the blog</a></p>
+    </div>
+    <div class="col"> 
+      <div class="p-image-container--16-9 u-hide--small is-highlighted">
+      <img class="p-image-container__image u-no-padding--top" style="width: 70%;" src="https://assets.ubuntu.com/v1/719e550b-Things%20to%20do.svg"  alt="" />
```

Comment:
There should be some spacing below the image as per design
<img width="374" alt="Screenshot 2024-09-03 at 12 14 29 PM" src="https://github.com/user-attachments/assets/82ea5883-5dd0-42a0-9420-927b49867491">